### PR TITLE
Update Polish translation

### PR DESCRIPTION
**Pull request by MrBartusek**
Add missing keys to Polish translation and update strings with invalid letter case

### CODE REVIEW

File: src/translations/pl.json [hunk 1]
```diff
@@ -1,10 +1,30 @@
 {
   "status": {
     "cleaning": "Sprzątanie",
+    "auto": "Automatyczne Sprzątnie",
+    "spot": "Sprzątanie Miejscowe",
+    "edge": "Sprzątanie Brzegowe",
```
Comment (MrBartusek) [inline src/translations/pl.json:6]: This's a wired way to translate it but I guess it's good enough

**Review by denysdovhan — CHANGES_REQUESTED** [review]
Could you please add those statuses to English version as well?

**Comment by MrBartusek** [comment]
@denysdovhan I'm not sure about which statuses are you exactly talking about? This is the exact list of messages from the `en.json` file at the time of creating this PR.

**Comment by denysdovhan** [comment]
Already in master. Resolved.

**Pull request closed without merging**